mathics/builtin/structure.py

```diff
@@ -291,24 +291,27 @@ def apply(self, p1, p2, evaluation):
 class OrderedQ(Builtin):
     """
     <dl>
-    <dt>'OrderedQ[$a$, $b$]'
+    <dt>'OrderedQ[{$a$, $b$}]'
         <dd>is 'True' if $a$ sorts before $b$ according to canonical
         ordering.
     </dl>
 
-    >> OrderedQ[a, b]
+    >> OrderedQ[{a, b}]
      = True
-    >> OrderedQ[b, a]
+    >> OrderedQ[{b, a}]
      = False
     """
 
-    def apply(self, e1, e2, evaluation):
-        "OrderedQ[e1_, e2_]"
+    def apply(self, expr, evaluation):
+        "OrderedQ[expr_]"
 
-        if e1 <= e2:
-            return SymbolTrue
-        else:
-            return SymbolFalse
+        for i in range(1, len(expr.leaves)):
```

Comment:
This idiom is not recommended in Python, could you replace it with [enumerate](https://realpython.com/python-enumerate/)?